docs/source/markdown/options/add-host.md

```diff
@@ -2,10 +2,28 @@
 ####>   podman build, create, farm build, pod create, run
 ####> If file is edited, make sure the changes
 ####> are applicable to all of those.
-#### **--add-host**=*host:ip*
+#### **--add-host**=*hostname[;hostname[;…]]*:*ip*|"host-gateway"
 
-Add a custom host-to-IP mapping (host:ip)
-Multiple hostnames for the same IP can be separated by semicolons.
+Adds a custom host-to-IP mapping to the <<container|pod>>'s `/etc/hosts` file.
 
-Add a line to /etc/hosts. The format is hostname:ip or hostname1;hostname2;hostname3:ip if you want to map multiple hostnames to the same ip without duplicating the --add-host parameter. The **--add-host**
-option can be set multiple times. Conflicts with the **--no-hosts** option.
+The option takes one or multiple semicolon-separated hostnames to be mapped to
+a single IPv4 or IPv6 address, separated by a colon. It can also be used to
+overwrite the IP addresses of hostnames Podman adds to `/etc/hosts` by default
+(also see the **--name** and **--hostname** options). This option can be
+specified multiple times to add additional mappings to `/etc/hosts`. It
+conflicts with the **--no-hosts** option.
+
+Instead of an IP address the special flag *host-gateway* can be given that
```

Comment:
Needs comma between `address` and `the`. And the "that resolves" is unclear. Suggestion:
>Instead of an IP address, the special flag *host-gateway* can be given. This resolves to an IP address that the container can use to connect to the host.